Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md

```diff
@@ -470,21 +472,22 @@ Note: To disable **assistant**, click on assistant again.
+1. **Display Name** - Optional: Specify a display name for the new experiment. There are no character or length restrictions for naming. If this field is left blank, Driverless AI will automatically generate a name for the experiment.
+2. **Dataset** - The name of the dataset being used to create an experiment
+3. **Rows** - Total number of rows 
+4. **Columns** - Total number of columns 
+5. [**Dropped Columns**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#dropped-columns) - Drop column(s) from your dataset that you don't want to use in the experiment
+6. [**Validation Dataset**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#validation-dataset) - Select the dataset you want to validate. This set will be used to validate parameters like models, features, etc.
+7. [**Test Dataset**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#test-dataset) - The dataset that will be used to test the model generated from the training dataset. It's not used during training of the model, and results are available at the end of the experiment.
 8. **Target column** -  What do you want to predict? 
-9. **Fold column** - The fold column is used to create the training and validation datasets so that all rows with the same Fold value will be in the same dataset. This column will be used if no validation dataset is provided and the model is trained using cross validation. 
-10. **Weight column** - Column that indicates the observation/isntance weight (a.k.a. sample or row weight), if applicable.  
-11. **Time Column**(OFF by default) - Provides a time order(timestamps for observations). Used when data has a high dependency on time (such as seasonality or trend), and you want to treat this problem as a time series problem. 
+9. [**Fold column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#fold-column) - The fold column is used to create the training and validation datasets so that all rows with the same Fold value will be in the same dataset. This column will be used if no validation dataset is provided and the model is trained using cross validation. 
+10. [**Weight column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#weight-column) - Column that indicates the observation/isntance weight (a.k.a. sample or row weight), if applicable.  
+11. [**Time Column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#time-column)(OFF by default) - Provides a time order(timestamps for observations). Used when data has a high dependency on time (such as seasonality or trend), and you want to treat this problem as a time series problem. 
 
 Continuing with our experiment:
 
+Type the name for your experiment `Titanic Classification Tutorial`
```

Comment:
change to: 

Name your experiment as follows: `Titanic Classification Tutorial`

Reply:
Done